android/PhysicalWeb/app/src/main/res/layout/list_item_nearby_beacon.xml

```diff
@@ -19,24 +19,35 @@
 
         <LinearLayout
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
+                android:layout_height="76dp"
                 android:orientation="vertical"
                 android:layout_weight="1"
-                android:gravity="start">
+                android:gravity="start"
+            android:weightSum="1">
             <ImageView
                     android:id="@+id/icon"
                     android:layout_width="24dp"
                     android:layout_height="24dp"
                     android:layout_marginTop="6dp"
                     android:contentDescription="@string/nearby_beacon_favicon_content_description">
             </ImageView>
+
+            <ImageView
+                android:id="@+id/star"
+                android:layout_width="30dp"
+                android:layout_height="30dp"
+                android:layout_marginTop="6dp"
+                android:contentDescription="@string/nearby_beacon_favicon_content_description"
```

Comment:
This should have its own contentDescription string.  Content description is mostly for accessibility (screen readers, eg) so pick a string that would be helpful to a vision-impaired user.  eg, "Toggle favorite"